calicoctl/commands/node/run.go

```diff
@@ -181,15 +194,23 @@ Description:
 
 	// Create a mapping of environment variables to values.
 	envs := map[string]string{
-		"HOSTNAME": name,
```

Comment:
We might want to keep both around for a little while, so that users can use the new calicoctl to launch old calico/node images?